build_deploy.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-make docker-login-and-push
+USER=$QUAY_USER make docker-login-and-push
```

Comment:
The only place where this is used in our repo is setting the `IMAGE_REPOSITORY` in `boilerplate/openshift/golang-osd-operator/project.mk`. Why don't we set that directly, this is quite misleading